psgo.go

```diff
@@ -43,13 +44,30 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+type IDMap struct {
```

Comment:
Sorry for not mentioning that before: Can we add docs to the new types and their members? It'll make browsing on godoc.org easier and let IDEs display the information.